distkeras/trainers.py

```diff
@@ -46,13 +46,16 @@ class Trainer(object):
               See: https://keras.io/objectives/
         worker_optimizer: string. String representing worker optimizer.
                           See https://keras.io/optimizers/
+        metrics: list of strings representing model evaluation metrics.
+                 See: https://keras.io/objectives/
     """
 
-    def __init__(self, keras_model, loss, worker_optimizer):
+    def __init__(self, keras_model, loss, worker_optimizer, metrics=["accuracy"]):
```

Comment:
Same as in workers.py, to preserve consistency, could you add "accuracy" as the default metric for all trainers?

You added it in the docs, but not in the code :)